README.md

```diff
@@ -1,77 +1,189 @@
 # packit.dev
 
-[packit.dev](https://packit.dev) website content
+Repository containing documentation and GH Actions used for deployment of the
+[packit.dev] website.
 
-## Clone
+### Installation
 
-This repository uses git submodules, so you have to `git clone --recurse-submodules` it.
-If you forget and realize later, just run:
+```
+$ yarn
+```
 
-    $ git submodule init themes/book
-    $ git submodule update
+### Local Development
 
-## Hugo
+```
+$ yarn start
+```
```

Comment:
This is not working for me, it tells me:

```[ERROR] Error: The docs folder does not exist for version "current". A docs folder is expected to be found at research```

Should we introduce a makefile target with other steps before this one? Am I missing something?

Reply:
Fixed by adding a makefile target `run-dev` that depends on `import` target that pull both research and deployment in